### PR TITLE
Note the signed commit requirement (Infra)

### PR DESCRIPTION
**Pull request by mz2**
Explain the signed commit requirement in documentation…
- in the contribution guide
- in the PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -10,9 +11,14 @@ Pick one of the following:
     - This includes any API change to checkbox-ng/checkbox-support
     - Changes to PXU grammar/field requirements
     - Breaking changes to dependencies in snaps (fwts upgrade for example)
-If your change is to providers it can only be (Infra, BugFix or New)
 
-Example Title: Fixed bugged behaviour of checkbox load config (Bugfix)
+If your change is to providers it can only be (Infra, BugFix or New).
+
+Signed commits are required.
+  - See CONTRIBUTING.md (https://github.com/canonical/checkbox/blob/main/CONTRIBUTING.md#signed-commits-required) for further instructions.
```
Comment (kissiel) [inline .github/pull_request_template.md:18]: missed `git add` for the CONTRIBUTING.md?

**Review by kissiel — CHANGES_REQUESTED** [review]
The template references non-existent section

File: .github/pull_request_template.md (hunk 1 above)
Comment (mz2) [reply .github/pull_request_template.md:18]: You have to have some belief on this one, it resolves once merged 😄 I couldn't think of a relative link to offer for this.

Comment (mz2) [reply .github/pull_request_template.md:18]: (The section is introduced in this same branch, with that `signed-commits-required` being a translation that GitHub's rendering does automatically for a `## Signed commits required`)

**Comment by mz2** [comment]
> The template references non-existent section

The template references a section introduced in the PR itself. I don't know that there is a way to review the correctness of that URL, I encourage you to join me on the ride on this one.

**Review by kissiel — APPROVED** [review]
Missed the contrib changes. LGTM

**Pull request merged**